array.test.js

```diff
@@ -0,0 +1,162 @@
+const _ = require("./array");
+
+test("Slice array", () => {

```

Comment:
It would be great to group your tests by methods with `describe` blocks:
* `describe('slice')`
    * `test('should slice array')`
* `describe('chunk')`
    * `test('should divide [1, 2, 3, 4] into groups of 2')`
    * `test('should divide [1, 2, 3, 4] into groups of 3')`
    * `test('should divide [1, 2, 3, 4] into groups of 7 (more than array length)')`
    * `test('should divide [1, 2, 3, 4] into groups of 1 if a number of chunks is not provided')`
    * `test('should not divide an empty array')`
etc.

And also I have to say that usually, we start a test name from the `should` word to reflect the behaviour.